src/Browscap/Data/Expander.php

```diff
@@ -43,6 +43,11 @@ class Expander
     private $logger = null;
 
     /**
+     * @var array
```

Comment:
OK no problem; same question, would it benefit a DTO to be used here to enforce structure?